src/main/java/trackr/model/task/TaskDeadline.java

```diff
@@ -0,0 +1,64 @@
+package trackr.model.task;
+
+import static java.util.Objects.requireNonNull;
+import static trackr.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a Task's deadline in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTaskDeadline(LocalDate)}
+ */
+public class TaskDeadline {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Task deadline should only contain numeric values in the format \"DD/MM/YYYY\""
+                    + "and it should not be blank";
```

Comment:
Since the value passed into the constructor is already a `LocalDate` instance, this means that the parsing to `LocalDate` already happened and resulted in something true and should not have this constraint.

Maybe could change the constructor to take in a string value and change isValidTaskDeadline to check if it is in the right syntax of `DD/MM/YYYY` then we have `taskDeadline = LocalDate.parse`